composites/Plugin/Shared/components/SynonymsInput.js

```diff
@@ -1,21 +1,30 @@
 import React from "react";
 import PropTypes from "prop-types";
 import uniqueId from "lodash/uniqueId";
+import styled from "styled-components";
 
 import { YoastInputContainer, YoastInputField, YoastInputLabel } from "./YoastInput";
-import HelpText from "../../Shared/components/HelpText";
+import { getRtlStyle } from "../../../../utils/helpers/styled-components";
 
-const SynonymsInput = ( { id, label, value, onChange, explanationText } ) => {
+const SynonymsFieldLabelContainer = styled.span`
+	margin-bottom: 0.5em;
+`;
+
+const StyledYoastInputLabel = styled( YoastInputLabel )`
+	display: inline-block;
+	margin-bottom: 0;
+	${ getRtlStyle( "margin-right: 4px", "margin-left: 4px" ) };
+`;
+
+const SynonymsInput = ( { id, label, labelSiblingElement, value, onChange } ) => {
 	return (
 		<YoastInputContainer>
-			<YoastInputLabel htmlFor={ id }>
-				{ label }
-			</YoastInputLabel>
-			{ explanationText && (
-				<HelpText>
-					{ explanationText }
-				</HelpText>
-			) }
+			<SynonymsFieldLabelContainer>
+				<StyledYoastInputLabel htmlFor={ id }>
+					{ label }
+				</StyledYoastInputLabel>
+				{ labelSiblingElement }
```

Comment:
Is this element ever going to be something else than a help link? If not, the name `labelSiblingElement` is too abstract.

Reply:
Thought to use this name for consistency with `KeywordInput` where the prop was previously named `postLabelElement`. No objections to change it, then it should be changed in `KeywordInput` too. Suggestions on the name welcome 🙂